framework/src/consumption/lib/redshift-serverless/redshift-serverless-namespace.ts

```diff
@@ -196,9 +196,21 @@ export class RedshiftServerlessNamespace extends TrackedConstruct {
         ],
         resources: [this.dataKey.keyArn, this.adminSecretKey.keyArn],
       }),
+      new PolicyStatement({
```

Comment:
Is this necessary? What if the service role is already created?

Reply:
This is required by Redshift serverless API even if the service linked role already exists